### DISCUSSION TITLE
How to stop truncation of items within TOC

### DISCUSSION THREAD

**Discussion by purpleclay** [squidfunk/mkdocs-material#5173] [Q&A] [answered]
I haven't updated my version of `mkdocs-material-insiders` for a while, but having just updated it, when I rebuild my site, all of the links within my TOC are truncated. I have looked around in the documentation on how to stop this from happening, but I can't find anything.

<img width="378" alt="Screenshot 2023-03-09 at 20 39 20" src="https://user-images.githubusercontent.com/106762954/224151965-e4e893e0-be92-4df2-85f3-d0eb0c44a5f7.png">

If I switch back to an older version, say `9.0.13-insiders-4.32.0`, this behaviour is not exhibited:

<img width="378" alt="Screenshot 2023-03-09 at 20 39 52" src="https://user-images.githubusercontent.com/106762954/224152070-7d04f009-e2f4-4ea6-b3ba-a56dd1b674fe.png">

**Comment by squidfunk** [comment, marked as answer]
That doesn't look intended. Could you please create an issue with a minimal reproduction?

**Reply by purpleclay** [reply]
Will do. Thanks for responding.
